README.md

```diff
@@ -111,6 +111,25 @@ $ sbt assembly
 
 After that find the uberjar at `target/scala-2.11/morganey.jar`
 
+### Custom sbt commands ###
+
+This project introduces the following custom sbt commands:
+
+- `sbt build`: Compiles the subprojects `kernel`, `macros` and
+  `stdlib` and copies the resulting jar-files into the target directory
+  of the root project.
+
+- `sbt buildModuleIndex`: Traverses the stdlib subproject and creates
+  a resource file containing a list of modules - the so called
+  [module-index]. This command is automatically run if you `sbt
```

Comment:
I think there's also a word-order error in my sentence. It should read "This command is run/executed automatically".